clarity/contracts/alex-vault.clar

```diff
@@ -15,6 +18,13 @@
 (define-constant ERR-MATH-CALL (err u2010))
 (define-constant ERR-INTERNAL-FUNCTION-CALL (err u1001))
 
+(define-map approved-contracts
+  { name: principal }
+  {
+    can-transfer: bool
+  }
+)
```

Comment:
This map can be simplified to save runtime cost.

```clojure
(define-map approved-contracts principal bool)
```

Reply:
This is sweet!